src/schema/users.ts

```diff
@@ -2687,17 +2598,6 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         }
 
         if (existingUserCompanyEmail.verified) {
-          // user has already verified this email, but want to verify work experience
-          const verifyUserCompany =
-            await completeVerificationForExperienceByUserCompany(
```

Comment:
Verifying a user's work experience is going to be based off a worker. Reference: https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/1816657933/RFC-146+-+Profile+4.0+Backend#Workers